libraries/botframework-connector/src/globals.ts

```diff
@@ -1,22 +0,0 @@
-/**
- * @module botbuilder
- */
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
- */
-
-// set FormData and Fetch as global functions
-function setGlobals() {
-    const env = (global || window) as any;
-    
-    if (!env.hasOwnProperty("FormData")) { 
-        env.FormData = require("form-data");
-    }
-
-    if (!env.hasOwnProperty("fetch")) {
-        env.fetch = require("node-fetch");
```

Comment:
No longer necessary using `import fetch from "cross-fetch"` which is isomorphic.